Backend/routes/api.php

```diff
@@ -22,8 +23,49 @@
 
 Route::post("/register", [AuthController::class, "register"]);
 Route::post("/login", [AuthController::class, "login"]);
-
+Route::get("/kebabs", [KebabController::class, "getAllKebabs"]);
 Route::middleware("auth:sanctum")->group(function (): void {
     Route::post("/logout", [AuthController::class, "logout"]);
     Route::get("/getCurrentUser", [UserController::class, "getCurrentUser"]);
+    Route::get("/fav/{id}", [UserController::class, "getUserFavorites"]);
+    Route::post("/addfav", [UserController::class, "addToFavorites"]);
+    Route::post("/remfav", [UserController::class, "removeFromFavorites"]);
```

Comment:
```Route::post("/remfav", [UserController::class, "removeFromFavorites"]);```

Reply:
maybe I'm wrong, but it's better to use delete for the remove action than post

Reply:
I left the POST even though we are deleting something from the database, I am deleting part of the json content, not the entire content, so I think it fits better than DELETE.